components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -2188,6 +2188,9 @@ type User struct {
 	MarkedDeleted bool   `json:"markedDeleted,omitempty"`
 	Name          string `json:"name,omitempty"`
 
+	// The ID of the Organization this user is owned by. If empty, the user is owned by the installation
+	OrganizationId string `json:"organizationId,omitempty"`
```

Comment:
```suggestion
	OrganizationId uuid.UUID `json:"organizationId,omitempty"`
```
ideally

Reply:
Historically we haven't been using it. I guess it's a question of type-safety, we can guide the caller towards the "right thing" by making it UUID.

I'll leave this up to you, if you don't want to depart from the current model, then feel free to leave it as string. But always sticking to how we've done things in the past doesn't always get us to a better place.